app/assets/javascripts/components/IncidentCard.js

```diff
@@ -9,43 +9,41 @@ class IncidentCard extends React.Component {
   render() {
     const {incident} = this.props;
     return (
-      <div style={{paddingTop: 60}}>
```

Comment:
The diff looks messy, but all this is doing is removing the `<div style={{paddingTop: 60}}>` wrapping everything; that shouldn't have been factored out of the caller (which still has it here: https://github.com/studentinsights/studentinsights/blob/master/app/assets/javascripts/student_profile/AttendanceDetails.js#L141).